src/main.rs

```diff
@@ -389,22 +411,31 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         DeployMethod::Local(local_path.to_owned())
     } else if let Some(release_channel) = matches.value_of("release_channel") {
         DeployMethod::ReleaseChannel(release_channel.to_owned())
+    } else if let Some(commit) = matches.value_of("commit") {
+        let github_user = matches
+            .value_of("github_user")
+            .expect("User should pass in --github-user <username>");
+        let repo_name = matches.value_of("repo_name").unwrap_or_default();
```

Comment:
nit: this should just be an `unwrap()` since a default value is specified -- the `_or_default()` part of this call actually calls into the base `Option` type, and not the clap default, so it would give the empty string, which would be bad for this case. So in your case, `unwrap()` is correct